code/modules/unit_tests/station_trait_tests.dm

```diff
@@ -0,0 +1,7 @@
+/datum/unit_test/station_traits/Run()
+	var/datum/station_trait/cybernetic_revolution/cyber_trait = new()
```

Comment:
```suggestion
	var/datum/station_trait/cybernetic_revolution/cyber_trait = allocate(/datum/station_trait/cybernetic_revolution)
```

This will automatically qdel it at the end.